recipes/libverto/all/conanfile.py

```diff
@@ -145,6 +134,14 @@ def generate(self):
             f"--with-libevent={yes_no_builtin(self.options.with_libevent)}",
             f"--with-tevent={yes_no_builtin(self.options.with_tevent)}",
             ])
+
+        env = tc.environment()
+        if is_msvc(self):
+            env.define("CC", "{} cl -nologo".format(unix_path(self.deps_user_info["automake"].compile))) 
+            env.define("CXX", "{} cl -nologo".format(unix_path(self.deps_user_info["automake"].compile))) 
+            env.define("LD", "{} link -nologo".format(unix_path(self.deps_user_info["automake"].compile))) 
+            env.define("AR", "{} lib".format(unix_path(self.deps_user_info["automake"].ar_lib))) 
+
         tc.generate()
```

Comment:
```suggestion
        tc.generate(env)
```

Reply:
Are you sure really need it? Recipe throw error with msvc, so we do no need this if we cannot build with msvc at all